hotspot/make/linux/makefiles/sa.make

```diff
@@ -59,9 +59,11 @@ SA_PROPERTIES = $(SA_CLASSDIR)/sa.properties
 
 # if $(AGENT_DIR) does not exist, we don't build SA
 # also, we don't build SA on Itanium or zero.
+# check for thread_db.h too (musl does not have it).
 
 all: 
-	if [ -d $(AGENT_DIR) -a "$(SRCARCH)" != "ia64" \
+if [ -d $(AGENT_DIR) -a -f /usr/include/thread_db.h \
```

Comment:
@calderast it's the indentation here for this makefile that needs to be fixed

Reply:
Fixed, thank you!